src/rpdk/core/contract/resource_client.py

```diff
@@ -147,6 +157,23 @@ def strategy(self):
         self._strategy = ResourceGenerator(schema).generate_schema_strategy(schema)
         return self._strategy
 
+    @property
+    def invalid_strategy(self):
+        # an empty strategy (i.e. false-y) is valid
+        if self._invalid_strategy is not None:
+            return self._invalid_strategy
+
+        # imported here to avoid hypothesis being loaded before pytest is loaded
+        from .resource_generator import ResourceGenerator
+
+        # make a copy so the original schema is never modified
+        schema = json.loads(json.dumps(self._schema))
+
+        self._invalid_strategy = ResourceGenerator(schema).generate_schema_strategy(
+            schema
+        )
+        return self._invalid_strategy
```

Comment:
this is really only invalid if a resource has read only properties? is there another way to create an invalid strategy that doesnt depend on this being the case in the schema?

Reply:
In case of update, a resource updated with createOnly property will also be invalid.